back/boxtribute_server/graph_ql/types.graphql

```diff
@@ -23,7 +23,7 @@ type UnboxedItemsCollection implements ItemsCollection {
 
 
 """
-Representation of a box storing items of a [`Product`]({{Types.Product}}) in a [`Location`]({{Types.Location}})
+Representation of a box storing items of a [`Product`]({{Types.Product}}) in a [`ClassicLocation`]({{Types.ClassicLocation}})
```

Comment:
I think a box can also be part of a distro spot, so the generic Location would be correct here?